setup.py

```diff
@@ -78,7 +78,7 @@
     "codecov",
     "coverage>=4.4",
     "flake8>=3.8.2",
-    "imageio>=2.5.0",
+    "imageio>=2.5.0,<=2.9.0",
```

Comment:
`imageio` is currently on 2.19 so adding this restriction will make our dependency a bit limiting. Why do we not support 2.9+?

Reply:
https://github.com/bentoml/BentoML/issues/2416

Reply:
> Unfortunately upgrading imageio is not possible since it breaks bentoml in a different way.

Do you know what breakages were referred to here? I think locking a non-major version upper bound is not ideal since it may introduce conflicts to user environments. If the failure is very niche, we can answer this question in our Github discussion. Wdyt?

Reply:
it seems that the 2.9.0 is released at 10 Jul 2020, which is around the time we are already working on migrating to 1.0. I think we then didn't follow up with updated changes from imageio. 

Sure we can answer it in GitHub actions, but I think a warning would deliver this faster.